# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/ipc/GPUProcessHost.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/ipc/GPUProcessHost.cpp
--- a/gfx/ipc/GPUProcessHost.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/ipc/GPUProcessHost.cpp	Fri May 24 11:32:54 2019 +0000
@@ -6,10 +6,10 @@
 
 #include "GPUProcessHost.h"
 #include "chrome/common/process_watcher.h"
-#include "gfxPrefs.h"
 #include "mozilla/gfx/Logging.h"
 #include "nsITimer.h"
 #include "mozilla/Preferences.h"
+#include "mozilla/StaticPrefs.h"
 #include "VRGPUChild.h"
 #include "ProcessUtils.h"
 
@@ -62,7 +62,7 @@
     return !!mGPUChild;
   }
 
-  int32_t timeoutMs = gfxPrefs::GPUProcessTimeoutMs();
+  int32_t timeoutMs = StaticPrefs::GPUProcessTimeoutMs();
 
   // If one of the following environment variables are set we can effectively
   // ignore the timeout - as we can guarantee the compositor process will be